nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/DeduplicateRecord.java

```diff
@@ -395,7 +391,7 @@ public void onScheduled(final ProcessContext context) {
     private FilterWrapper getFilter(ProcessContext context) {
         if (useInMemoryStrategy) {
             boolean useHashSet = context.getProperty(FILTER_TYPE).getValue()
-                    .equals(context.getProperty(HASH_SET_VALUE.getValue()));
+                    .equals(context.getProperty(HASH_SET_VALUE.getValue()).getValue());
```

Comment:
Is the additional `getValue()` call necessary?

Reply:
This was flagged by the IntelliJ compiler, which believes the result of the comparator is always `false`. So I suppose it's not strictly necessary, it just keeps the compiler happy.